spot-vaults/contracts/charm/UsdcSpotManager.sol

```diff
@@ -37,6 +48,17 @@ contract UsdcSpotManager is Ownable {
     /// @notice The recorded deviation factor at the time of the last successful rebalance operation.
     uint256 public prevDeviation;
 
+    /// @notice The lower and upper deviation factor within which
+    ///         SPOT's price is considered to be in the active zone.
```

Comment:
Looking at the logic further down, this comment isn't exactly accurate.

This zone determines the max deviation where concentrated liquidity bands can be placed. So, the deviation factor range is smaller than this range. This distinction will be important for setting the configuration correctly.

Reply:
updated the comment